src/js/grapl-cdk/extract-grapl-deployment-artifacts.sh

```diff
@@ -2,18 +2,29 @@
 
 if [ -z "$VERSION" ]
 then
-  echo "Please set VERSION= (it's same as TAG= for dobi)"
+  echo "Please set VERSION="
   exit 1
 fi
 
+export GRAPLROOT=$(realpath ../../..)
+export ZIPS_DIR=$(realpath zips)
+
```

Comment:
my bash-fu is not phenomenal, and I overuse export VAR=thing when I could drop the export